.travis.yml

```diff
@@ -12,14 +12,11 @@ jobs:
     - env: TEST="scalafmt"
       script: ./bin/scalafmt --test
     - env: TEST="2.11"
-      script:
-        - sbt ++2.11.12 lsp4s/test
-        - sbt ++2.11.12 jsonrpc/test
-    - env: TEST="sbt test"
-      script:
-        - sbt test
+      script: sbt ++2.11.12 test
+    - env: TEST="2.12"
+      script: sbt test
     - stage: release
-      script: sbt ci-release
+      script: travis_wait 60 sbt ci-release
```

Comment:
Why is this wait needed?

Reply:
sonatypeReleaseAll can be so slow that Travis kills the release job due to "no activity". Happened last week in the lsp4s repo.

Reply:
Ok! I misunderstood what the function does: it doesn't wait, it sets a different timeout. 👍